app/app.go

```diff
@@ -98,6 +98,20 @@ func (a *App) InitServer() {
 			})
 		}
 		a.srv.RunJobs()
+		if a.IsLeader() {
```

Comment:
This whole logic should be moved inside a separate function `runDNDStatusExpireJob(a)` and called from inside `a.Srv().runjobs` like above.

Reply:
runJobs won't have app receiver available so can't call from inside it